src/UWP/ArcGISRuntime.UWP.Viewer/SourceCodeViewer.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<UserControl x:Class="ArcGISRuntime.UWP.Viewer.SourceCode"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:models="clr-namespace:ArcGISRuntime.Samples.Shared.Models"
+             mc:Ignorable="d"
+             Background="{StaticResource PrimaryBackgroundColor}">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
```

Comment:
I recommend reviewing this file for opportunities to simplify. I believe the column definitions can be removed, as well as setting width and height to auto on the ComboBox and WebView. There may be other simplifications that can be made.